aya/src/obj/mod.rs

```diff
@@ -222,6 +224,7 @@ impl FromStr for ProgramSection {
                     })
                 }
             },
+            "cgroup/sysctl" => CgroupSysctl { name },
```

Comment:
As I see for another mappings (cgroup/skb, cgroup => skb), I think the mapping for "cgroup => sysctl" should also be implemented (for now is only cgroup/sysctl)

Reply:
Yes, makes sense. Updated. Thank you!